app/models/workshop.rb

```diff
@@ -1,2 +1,3 @@
 class Workshop < ApplicationRecord
+  belongs_to :event
```

Comment:
The workshop is the topic that we teach during an event. so a workshop has many events. and an event belongs to a workshop